data/Biocontainers debian based image/biocontainers.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
this one looks wired, from the ID

Reply:
hum, base images should not have been included